xml/System.ServiceModel.Syndication/XmlDateTimeData.xml

```diff
@@ -86,9 +92,8 @@
         <ReturnType>System.Xml.XmlQualifiedName</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <summary>Gets the ElementQualifiedName property.</summary>
```

Comment:
The `ElementQualifiedName` should be formatted as code.

Reply:
@BillWagner do you mean a full <see cref="..." /> or do you mean <c>...</c>?